src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -23,7 +23,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Containers.ContainerRegistry" Version="1.1.0" />
-    <PackageReference Include="Azure.Identity" Version="1.9.0" />
+    <PackageReference Include="Azure.Identity" Version="1.10.4" />
```

Comment:
We can't upgrade this - see https://github.com/Azure/azure-sdk-for-net/issues/39377